src/index.js

```diff
@@ -0,0 +1,61 @@
+// eslint-disable 
+import './style.css';
```

Comment:
- [ ] I noticed your linters are failing and this is because you disabled the linters. Erace the commented lines on `src/index.js`